core/src/main/java/org/mapstruct/BeanMapping.java

```diff
@@ -18,7 +18,9 @@
 /**
  * Configures the mapping between two bean types.
  * <p>
- * Either {@link #resultType()}, {@link #qualifiedBy()} or {@link #nullValueMappingStrategy()} must be specified.
+ * When source or the target is of type {@code Iterable} or {@code Stream}, it treats the target as a simple pojo and
```

Comment:
some textual stuff: I would use "handles" instead of "treats".. (a bit nitpicking, I agree)..